### PR TITLE
feat: /data/kafka refresh

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied changes as per [doc](https://docs.google.com/document/d/1bfo97XdFdKlzrMcMoRxiTdzt_bPVndx6j55IPsapuAE/edit) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1399-1827&node-type=canvas&m=dev)
- Drive by:
  - Removed placeholder text from corresponding form

## QA

- View the site locally in your web browser at: https://canonical-com-1392.demos.haus/data/kafka
- Check against doc and design 
  - Be sure to check smaller screens as well

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15062

### CODE REVIEW

File: templates/data/kafka.html [hunk 1]
```diff
@@ -18,314 +16,393 @@
...
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-          </div>
-        </div>
+      <p>Get comprehensive, 24/7 or weekday support for Charmed Kafka, covering the entire solution for up to 10 years.</p>
+      <div class="p-cta-block">
+        <a class="p-button" href="">Learn more</a>
```
Comment (mtruj013) [inline templates/data/kafka.html:213]: CTA is still wip and will 404 for now, see discussion [here](https://docs.google.com/document/d/1bfo97XdFdKlzrMcMoRxiTdzt_bPVndx6j55IPsapuAE/edit?disco=AAABLcLNUYI)

**Comment by Sophie-32** [comment]
- [ ] Add better language for learn more (I'm ok with doing that ourselves, I updated the copydoc)

Screen Shot 2024-10-09 at 2 29 37 PM

**Comment by lyubomir-popov** [comment]
- can we please add --on large here:

image

- logo in hero needs replacing. 
- logos in logo section wall need replacing
(assets here https://assets.ubuntu.com/manager?tag=kafkapageassets)

The rest looks great, thanks @mtruj013

**Comment by Sophie-32** [comment]
Much better without the placeholder text. While we're here, Juan found some small typos in the demo. 

- [ ] Remove the "M" at the end of the "Simple per node subscriptionM" heading
- [ ]  There's a typo in the "Easy to deply and manage" H2, should be "deploy"
- [ ]  Add a chevron to "[Get started with Charmed Kafka ›](https://canonical-com-1392.demos.haus/data/docs/kafka/iaas)"

**Comment by mtruj013** [comment]
@Sophie-32 done!

**Comment by mtruj013** [comment]
@mattea-turic could you have a look at Britney's comments please?

**Comment by mattea-turic** [comment]
> @mattea-turic could you have a look at Britney's comments please?

I agree with britney, there should be padding for each section. I took a look at the live page and it looks good.

There's a little too much padding here, but I think that might be due to the button?
Screenshot 2024-10-16 at 10 18 03

**Review by britneywwc — APPROVED** [review]
Thanks for making the changes, looks great! 🥳

**Pull request merged**